### PR TITLE
DO-2237 Remake PEP detailed view in DO platform

### PR DESCRIPTION
**Pull request by EmmaSamoilova**

### CODE REVIEW

**Review by roman-tiukh — APPROVED** [review]
PR is big, but looks good

File: src/locales/en/translation.json [hunk 1]
```diff
@@ -207,19 +224,23 @@
   "isDead": "Is Dead",
   "itn": "ITN",
   "koatuu-fourth-level": "KOATUU",
+  "land": "Land",
   "lastName": "Last name",
   "lastPlaceOfWork": "Last place of work",
   "lastPosition": "Last position",
   "lastUpdated": "Last updated",
   "legalEntity": "Legal entity",
+  "liability": "Finanvial liabilities",
```
Comment (Andrii17s) [inline src/locales/en/translation.json:233]: May be "Financial liabilities"?

**Review by Andrii17s — CHANGES_REQUESTED** [review]

File: src/locales/en/translation.json (hunk 1 above)
Comment (EmmaSamoilova) [reply src/locales/en/translation.json:233]: Yes, thank you

File: src/components/pages/SystemPage.jsx [hunk 2]
```diff
@@ -79,7 +79,7 @@ const SystemPage = (props) => {
         <Nav>
           {menu}
         </Nav>
-        <div className="content overflow-x-auto pb-0">
+        <div className="content pb-0">
```
Comment (roman-tiukh) [inline src/components/pages/SystemPage.jsx:82]: this change did not break any page on the platform?

File: src/components/pages/datasets/pep/PepDetail.jsx [hunk 3]
```diff
@@ -1,237 +1,416 @@
...
+  const liabilityRef = useRef();
+  const realEstateRef = useRef();
+  const carRef = useRef();
+  const otherNamesRef = useRef();
+  const additionalInfoRef = useRef();
+  const relatedPersonRef = useRef();
+  const giftRef = useRef();
+  const expendituresRef = useRef();
+  const ASYNCBLOCK = 'ASYNCBLOCK';
+  const INFOBLOCK = 'INFOBLOCK';
```
Comment (roman-tiukh) [inline src/components/pages/datasets/pep/PepDetail.jsx:67]: this can be moved in to const.js file

**Review by roman-tiukh — DISMISSED** [review]

File: src/components/pages/datasets/pep/PepDetail.jsx (hunk 3 above)
Comment (EmmaSamoilova) [reply src/components/pages/datasets/pep/PepDetail.jsx]: done

File: src/components/pages/SystemPage.jsx (hunk 2 above)
Comment (EmmaSamoilova) [reply src/components/pages/SystemPage.jsx:82]: No, I checked

**Review by roman-tiukh — APPROVED** [review]

**Review by AlenaYanish — APPROVED** [review]

**Review by Andrii17s — APPROVED** [review]

**Pull request merged**